src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.IOException;
+
+public class Application {
+    public static void main(String[] args) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        File reportFile = new File("report.csv");
+        try {
+            reportFile.createNewFile();
+            workWithFile.getStatistic("orange.csv", "report.csv");
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create file", e);
+        }
```

Comment:
You're catching a general IOException to handle a file creation failure, which is good. However, the exception message "Can't create file" is not very informative. It would be better to include the file name in the exception message to make it clear which file couldn't be created.